docker/baseimage/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# syntax=docker/dockerfile:1
+ARG SOURCES_DIR=/usr/src/ipa
+FROM rust:latest as builder
+ARG SOURCES_DIR
+LABEL maintainer="akoshelev"
+
+# Prepare helper binaries
+WORKDIR "$SOURCES_DIR"
+COPY . .
```

Comment:
this may be ok in this place, but this basically transfers the whole directory as docker context, so it may take a while if there are build artifacts

Reply:
Lets reduce the size in later PRs